apps/condo/domains/user/oidc/adapter/OidcModelClientAdapter.js

```diff
@@ -20,6 +20,7 @@ class OidcModelClientAdapter {
      *
      */
     constructor (name, context) {
+        if (name !== 'Client') throw new Error('OidcModelClientAdapter can work only with OidcClient schema')
```

Comment:
It's ok, but only the `Client` type is stored by this adapter

<img width="895" alt="image" src="https://user-images.githubusercontent.com/25384290/222879591-4097e22b-cebd-41bd-9eb8-c0d52f704d48.png">

apps/condo/domains/user/oidc/adapter/AdapterFactory.js

Reply:
it's just an extra guard inside the class. I don't think that it's redundant. Bore guards = less runtime errors